backend/Api/Controllers/v1/AreaController.cs

```diff
@@ -25,8 +25,15 @@ public AreaController(IMediator mediator, IMapper mapper)
     [HttpGet]
     public async Task<ActionResult<IList<Area>>> GetAllAreas()
     {
-        var result = await _mediator.Send(new GetAllAreasQuery());
-        return Ok(result);
+        try
+        {
+            var result = await _mediator.Send(new GetAllAreasQuery());
+            return Ok(result); 
+        }
+        catch (Exception ex)
+        {
+            return BadRequest(ex.Message);
```

Comment:
Super smart att ha med try/catch i entry point. Bra jobbat!
Osäker på om BadRequest är rätt att använda här, vi behåller det så här tills vidare så kan jag undersöka detta.
Det kan vara bra att även logga felet, exempelvis:  _logger.Error("Error in GetAllAreas: {message}", ex).

Approvar denna PR men om du bara kan lägga till loggningen innan merge :D
Gäller självklart i samtliga endpoints 👍

Reply:
Bra tillägg! Skadar inte att vara lite mer specifik heller